modin/pandas/series.py

```diff
@@ -1544,8 +1567,13 @@ def repeat(self, repeats, axis=None):  # noqa: PR01, RT01, D200
 
         return self.__constructor__(query_compiler=self._query_compiler.repeat(repeats))
 
-    def _reset_index(
-        self, level, drop, name, inplace, allow_duplicates
+    def reset_index(
```

Comment:
@mvashishtha why we still don't use `@_inherit_docstrings` for all methods in Series? Maybe you know.

Reply:
In fact, this decorator applies to the entire class and therefore to all public methods. I was confused by the fact that this function has a redefined docstring. I guess we need to remove them in a separate pull request.

Reply:
https://github.com/modin-project/modin/issues/5286